public-ui/src/app/data-browser/views/survey-chart/survey-chart.component.css

```diff
@@ -16,3 +16,7 @@
 .survey-chart-choice:hover {
     color: #262262;
 }
+
+.chart {
+    z-index: 17000;
```

Comment:
these z-indexs are getting wild. Im going to make a PR after this goes in to level them out

Reply:
Totally agree.